### PR TITLE
meson: configure spdk/dpdk to build on generic arm64 (Apple Silicon)

### PR DESCRIPTION
**Pull request by Baekalfen**
Signed-off-by: Mads Ynddal <m.ynddal@samsung.com>

### CODE REVIEW

File: subprojects/packagefiles/spdk/meson.build [hunk 1]
```diff
@@ -20,14 +20,28 @@ if get_option('build_subprojects') and not fs.exists('build' / 'lib' / 'src' / '
...
+
   message('Building ..')
   run_command([
       find_program(host_machine.system() == 'freebsd' ? 'gmake' : 'make'),
       '-j',
       cpu_count
     ],
     capture: true,
-    check: true
+    check: true,
```
Comment (safl) [inline subprojects/packagefiles/spdk/meson.build:43]: Shouldn't this still check error-value?

File: subprojects/packagefiles/spdk/meson.build [hunk 2]
```diff
@@ -20,14 +20,28 @@ if get_option('build_subprojects') and not fs.exists('build' / 'lib' / 'src' / '
     check: true,
   ).stdout().strip()
 
+  message('Checking if generic platform (Apple Silicon)')
+  if target_machine.cpu_family() == 'aarch64'
```
Comment (safl) [inline subprojects/packagefiles/spdk/meson.build:24]: How about:
```
is_platform_generic = target_machine.cpu_family() == 'aarch64' and run_command([
    import('python').find_installation('python3'),
    '-c',
    'print(open("/sys/devices/system/cpu/cpu0/regs/identification/midr_el1").read() == "")'
  ],
  check: true,
).stdout().strip() == 'True'
```

?

**Review by safl — COMMENTED** [review]
Great! Two questions/comments inline.

File: subprojects/packagefiles/spdk/meson.build (hunk 1 above)
Comment (safl) [reply subprojects/packagefiles/spdk/meson.build:43]: Oh, it still does, not there is just ",". Misread, nevermind.

File: subprojects/packagefiles/spdk/meson.build (hunk 2 above)
Comment (Baekalfen) [reply subprojects/packagefiles/spdk/meson.build]: You're right. I originally intended to return the string, that's why. Fixed now.

**Review by safl — APPROVED** [review]
Thanks!

**Pull request merged**